### PR TITLE
Update documentation for publishing/releasing

### PR DESCRIPTION
**Pull request by dslmeinte**

### CODE REVIEW

**Review by joswarmer — APPROVED** [review]
Looks good

File: readme.md [hunk 1]
```diff
@@ -115,24 +115,63 @@ The example assumes we merge `mps2021.1` into `mps2021.2`.
 5. Run all tests
 6. Update build model
 
-## Publishing and Releasing
 
-Run:
+## Publishing and releasing
```
Comment (enikao) [inline readme.md]: All the other headings are in all-caps variant, why change?

File: readme.md [hunk 2]
```diff
@@ -115,24 +115,63 @@ The example assumes we merge `mps2021.1` into `mps2021.2`.
...
-```
+1. Register on Sonatype.
+	You'll need the corresponding credentials later on, but note that these are different from the user token you'll also need (to create).
+2. Ask to be added to the list of users authorized to release under `io.lionweb`.
+  <!-- **Check**: is this still accurate? -->
+
+2. Create an _user token_ for Sonatype:
+  * Log in on [https://s01.oss.sonatype.org/](https://s01.oss.sonatype.org/) — this is **not** the same site as [Maven Central](https://central.sonatype.com/)
+  * Go to your profile (dropdown under your user name in the top-right corner)
+  * Click the “Generate User Token” (or “Access User Token” if you've already made one)
```
Comment (enikao) [inline readme.md:137]: I think you've missed the dropdown to switch to tokens

File: readme.md [hunk 3]
```diff
@@ -115,24 +115,63 @@ The example assumes we merge `mps2021.1` into `mps2021.2`.
...
+  * Set the username and password shown as the values of the `ossrhUsername` and `ossrhPassword` properties in `~/.gradle/gradle.properties`:
 
-In order to be able to relase you need to register on sonatype and then asked to be added to the list of users authorized to relase under `io.lionweb`.
+    ```shell
```
Comment (enikao) [inline readme.md]: this is not `shell`, more like `properties`

File: readme.md [hunk 4]
```diff
@@ -115,24 +115,63 @@ The example assumes we merge `mps2021.1` into `mps2021.2`.
...
+`./gradlew publish`
+
+to publish a snapshot version, or
+
+`./gradlew release`
+
+to release an official version.
+The `release` task involves manual input, which should consist of pressing Enter every of the **two** times it's requested.
+
+**Note that you must only publish/release from a branch `mps2021.<n>` where `<n>` = 1, 2, or 3.**
```
Comment (enikao) [inline readme.md]: Shall we keep this more generic? Sooner or later we will have newer MPS version branches

Comment (dslmeinte) [reply readme.md]: Done.

File: readme.md (hunk 2 above)
Comment (dslmeinte) [reply readme.md:137]: Added.

File: readme.md (hunk 1 above)
Comment (dslmeinte) [reply readme.md]: Changed it back.

**Pull request merged**